monitoring/src/grafana.ts

```diff
@@ -4,12 +4,9 @@ import * as pulumi from '@pulumi/pulumi'
 export interface deploymentArgs {
   namespace: pulumi.Input<string>
   domain: string
-  githubOrg: string
-  githubOauthID: string
-  githubOauthSecret: string
 }
 
-export class Deployment extends pulumi.ComponentResource {
+export class Ingress extends pulumi.ComponentResource {
```

Comment:
we are going to need to add host routing rules to the ingress route for `https://monitoring.shapeshift.com` to work correctly (assuming that is how the root url is intended to work in the kube-prometheus stack, not sure what that rootUrl is actually doing).

Reply:
We probably could skip it - I know that tool allows to configure nginx ingress but with Traefic I'd rather rely on our stuff. Will patch the ingress here

Reply:
I've ditched the domain config in yaml, let's add the DNS entry and we should be good